x-pack/plugins/monitoring/server/config.test.ts

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { createConfig, configSchema } from './config';
```

Comment:
Suggestion: if you mock the `fs` module... (see next comment)

```suggestion
import { createConfig, configSchema } from './config';

jest.mock('fs', () => ({
  ...jest.requireActual('fs'),
  readFileSync: jest.fn().mockImplementation((path: string) => `contents-of-${path}`),
}));
```

Reply:
Great suggestion!